device.go

```diff
@@ -122,15 +122,6 @@ func getPCIDeviceName(s *sandbox, pciID string) (string, error) {
 		}
 	}
 
-	// Check if the PCI path is present before we setup the pci device map.
-	_, err = os.Stat(filepath.Join(rootBusPath, pciAddr))
-	if err == nil {
```

Comment:
What's missing is `s.Unlock()` here. Could you please add it here instead of reverting the whole commit? It does fix a situation that is not handled in the original code.

Reply:
@bergwolf See my comment in the commit message for this. We should be taking out this check.
For eg, checking for /sys/devices/pci0000\:00/0000:00:04.0 is not sufficient, the device is fully attached when virtio# directory under that path  (/sys/devices/pci0000\:00/0000:00:04.0/virtio3/device). is created which we can only deduce with the uevent.